greenlight/tests/functional/keyboard_shortcuts/test_browser_window.py

```diff
@@ -39,9 +39,11 @@ def test_search_field(self):
 
         keys = [self.keys.ACCEL]
         if self.marionette.session_capabilities['platformName'] == 'LINUX':
-            keys.append(self.browser.get_localized_entity('searchFocusUnix.commandkey'))
+            keys.append(self.browser.get_localized_entity(
+                        'searchFocusUnix.commandkey'))
```

Comment:
I don't think it's acceptable to attach a linter to our CI that requires us to re-write our code in less readable ways. @whimboo, can we set the limit to 100 or disable this check?